src/lib.rs

```diff
@@ -170,7 +188,6 @@ pub fn from_path(p: &str) -> Result<String> {
         .pretty_print(&map)
         .map_err(|e| raw_to_parse_error(&map, *e))
 }
-
```

Comment:
I think this is extra.

Reply:
It is exactly the same as `from_string` so I don't think it's extra

Reply:
Sorry, I meant an extra line was deleted, we should have one line.